src/detect-engine-state.c

```diff
@@ -89,9 +89,9 @@ static DeStateStore *DeStateStoreAlloc(void)
 }
 
 #ifdef DEBUG_VALIDATION
-static int DeStateSearchState(DetectEngineState *state, uint8_t direction, SigIntId num)
+static int DeStateSearchState(DetectEngineState *state, uint8_t dsi, SigIntId num)
```

Comment:
what is `dsi`?

Reply:
Dir State Index...

Reply:
just `i`/`idx`/`direction` would be clearer, `dsi` has me wondering about the deeper meaning